app/models/container_image.rb

```diff
@@ -45,9 +46,10 @@ def operating_system=(value)
   end
 
   def docker_id
-    if image_ref.start_with?(DOCKER_IMAGE_PREFIX)
-      return image_ref[DOCKER_IMAGE_PREFIX.length..-1]
+    [DOCKER_IMAGE_PREFIX, DOCKER_PULLABLE_PREFIX].each do |prefix|
+      return image_ref[prefix.length..-1] if image_ref.start_with?(prefix)
```

Comment:
@enoodle I think we should add a new `docker_pullable_id` as well (which is returning only if prefix is `docker-pullable://`). Not for this PR though.

Also remember that this part (`docker_pullable_id`) may be very much relevant from the smart state analysis side (cc @moolitayer )